components/brave_rewards/browser/database/database_util.cc

```diff
@@ -9,6 +9,18 @@
 
 namespace brave_rewards {
 
+bool DropTable(
+    sql::Database* db,
+    const std::string& table_name) {
+  DCHECK(!table_name.empty());
```

Comment:
We should check `db` as well - I'm guessing this pattern (of not checking) is well established - but I'd recommend adopting it going forward.

Reply:
done